### PR TITLE
Modified scripts Mar 16

### PR DESCRIPTION
**Pull request by manjunath-grl**
Fixes Issue [502](https://github.com/CHIP-Specifications/chip-test-scripts/issues/502), [506](https://github.com/CHIP-Specifications/chip-test-scripts/issues/506), [510](https://github.com/CHIP-Specifications/chip-test-scripts/issues/510), [503](https://github.com/CHIP-Specifications/chip-test-scripts/issues/503), [511](https://github.com/CHIP-Specifications/chip-test-scripts/issues/511),[514](https://github.com/CHIP-Specifications/chip-test-scripts/issues/514), [513](https://github.com/CHIP-Specifications/chip-test-scripts/issues/513), [737](https://github.com/CHIP-Specifications/chip-certification-tool/issues/737), [735](https://github.com/CHIP-Specifications/chip-certification-tool/issues/735), [730](https://github.com/CHIP-Specifications/chip-certification-tool/issues/730), [726](https://github.com/CHIP-Specifications/chip-certification-tool/issues/726), [714](https://github.com/CHIP-Specifications/chip-certification-tool/issues/714), [717](https://github.com/CHIP-Specifications/chip-certification-tool/issues/717), [718](https://github.com/CHIP-Specifications/chip-certification-tool/issues/718), [741](https://github.com/CHIP-Specifications/chip-certification-tool/issues/741), [742](https://github.com/CHIP-Specifications/chip-certification-tool/issues/742), [746](https://github.com/CHIP-Specifications/chip-certification-tool/issues/746),[745](https://github.com/CHIP-Specifications/chip-certification-tool/issues/745)

Modified tests:
TC-LVL-3.1
TC-DD-1.11
TC-DD-2.1
TC-OPCREDS-3.6
TC-BRBINFO-2.3
TC-DRLK-2.7
TC-DGTHREAD-1.1
TC-DRLK-2.7
TC-DA-1.2
TC-DRLK-2.11
TC-ACL-2.7
TC-ACL-2.8
TC-ACL-2.10
TC-CADMIN-1.3
TC-CADMIN-1.4
TC-CADMIN-1.5
TC-CADMIN-1.6
TC-CADMIN-1.10
TC-CADMIN-1.13
TC-CADMIN-1.15
TC-CADMIN-1.16
TC-ACL-2.4
TC-ACL-2.9
TC-CADMIN-1.11
TC-CADMIN-1.12
TC-CADMIN-1.17
TC-CADMIN-1.18
TC-CADMIN-1.22
TC-CADMIN-1.24
TC-CADMIN-1.7
TC-CADMIN-1.9
TC-CC-5.1
TC-DRLK-2_11
TC-DRLK-2_9
TC-SC-5.1
TC-SC-5.2
TC_BR_4
TC_CC_2_1
TC_CC_7_5
TC-CNET-4-12
TC-CNET-4-14
TC-CNET-4-17
TC-CNET-4-18
TC-CNET-4-19
TC-DD-1-11
TC-DESC-1-1
TC-DGWIFI-3-2
TC-DRLK-2-11
TC-DRLK-2-3
TC-DRLK-2-9
TC-OCC-2-2-Simulated
TC-OCC-2-4-Simulated
TC-PCC-2-2
TC-PCC-2-3
TC-PCC-2-4
TC-SC-6-1
TC-ULABEL-3-1-Simulated
TC-BIND-2-1
TC-BIND-2-3
TC-BRBINFO-2-3
TC-BR-1
TC-BR-3
TC-BR-4
TC-CADMIN-1-1
TC-CADMIN-1-2
TC-CNET-4-15
TC-CONTENTLAUNCHER-10-5
TC-DA-1-4
TC-DD-1-5
TC-DGGEN-2-3
TC-DGTHREAD-2-1
TC-DRLK-2-11
TC-DRLK-2-9
TC-DRLK-3-1
TC-DRLK-3-2
TC-MC-11-1
TC-MC-11-2
TC-MOD-1-3
TC-MOD-3-1
TC-CC-2-2
TC-CC-3-4
TC-G-2-3
TC-MOD-2-1
TC-PS-2-2
TC-SWTCH-2-2
TC-SWTCH-3-2

Scripts reverted back to Manual testcases.
TC-ACT-3.1
TC-BOOL-3.1
TC-ALOGIN-12.1
TC-BRBINFO-1.1
TC-BRBINFO-2.1
TC-FLW-3.1
TC-I-3.1
TC-PRS-3.1
TC-PSCFG-3.1
TC-PS-3.1
TC-SWTCH-3.1
TC-WAKEONLAN-4.1
Added auto generated files

Tested

Execution logs:
[exe_log.txt](https://github.com/project-chip/connectedhomeip/files/10992640/exe_log.txt)

### CODE REVIEW

**Comment by raju-apple** [comment]
For context , the roll back of tests was necessary since there weren't any DUTs which could validate the specified automated tests during SVE

**Review by raju-apple — APPROVED** [review]

File: src/app/tests/suites/certification/Test_TC_BR_4.yaml [hunk 1]
```diff
@@ -6019,7 +6154,7 @@ tests:
             [1663916825.767283][1588:1588] CHIP:DMG:                                 DataVersion = 0x6bd99dc9,
             [1663916825.767309][1588:1588] CHIP:DMG:                                 AttributePathIB =
             [1663916825.767388][1588:1588] CHIP:DMG:                                 {
-            [1663916825.767423][1588:1588] CHIP:DMG:                                         Endpoint = 0xd,
+            [1663916825.767423][1588:1588] CHIP:DMG:                                         Endpoint = 0x3,
             [1663916825.767456][1588:1588] CHIP:DMG:                                         Cluster = 0x1d,
```
Comment (Survensa) [inline src/app/tests/suites/certification/Test_TC_BR_4.yaml:6158]: Should be [1663916825.767456][1588:1588] CHIP:DMG: Cluster = 0x39,

File: src/app/tests/suites/certification/Test_TC_BR_4.yaml [hunk 2]
```diff
@@ -6019,7 +6154,7 @@ tests:
...
             [1663916825.767502][1588:1588] CHIP:DMG:                                         Attribute = 0x0000_0003,
```
Comment (Survensa) [inline src/app/tests/suites/certification/Test_TC_BR_4.yaml:6159]: Should be [1663916825.767502][1588:1588] CHIP:DMG: Attribute = 0x0000_0005,

File: src/app/tests/suites/certification/Test_TC_G_2_3.yaml [hunk 3]
```diff
@@ -589,7 +589,7 @@ tests:
       verification: |
           ./chip-tool groupkeymanagement read group-table 1 0
 
-          Verify the "grouptable" does not have GroupName  Gp9 on the TH(Chip-tool)  Log and below is the sample log provided for the raspi platform:
+          Verify the "grouptable" does not have GroupName  gp9 on the TH(Chip-tool)  Log and below is the sample log provided for the raspi platform:
```
Comment (Survensa) [inline src/app/tests/suites/certification/Test_TC_G_2_3.yaml:592]: It should be Verify the "grouptable" does not have GroupName gp5 on the TH(Chip-tool) Log and below is the sample log provided for the raspi platform:

File: src/app/tests/suites/certification/Test_TC_BR_1.yaml [hunk 4]
```diff
@@ -326,9 +315,8 @@ tests:
 
           ./chip-tool  descriptor read device-type-list 1 4
 
+          Verify the DeviceTypeList entries with endpoint 1  On TH(chip-tool) Log
```
Comment (Survensa) [inline src/app/tests/suites/certification/Test_TC_BR_1.yaml:318]: It should be 

Verify the DeviceTypeList entries with endpoint 4 On TH(chip-tool) Log

**Review by Survensa — APPROVED** [review]

**Review by krypton36 — APPROVED** [review]

**Review by andy31415 — APPROVED** [review]

**Pull request merged**